Fluid.MvcSample/Startup.cs

```diff
@@ -12,14 +12,14 @@ public class Startup
         static Startup()
         {
             TemplateContext.GlobalMemberAccessStrategy.Register<Person>();
-            FluidViewTemplate.Factory.RegisterTag<MyTag>("mytag");
         }
 
         // This method gets called by the runtime. Use this method to add services to the container.
         // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc().AddFluid();
+            services.AddMvc().AddFluid()
```

Comment:
What about have the options inside the call to AddFluid, and also only having `Add` that would accept either a tag or a block ?